code/modules/clothing/chameleon.dm

```diff
@@ -247,6 +250,39 @@
 		return
 	random_look(owner)
 
+/datum/action/item_action/chameleon/change/proc/apply_job_data(datum/job/job_datum)
+	return
+
+/datum/action/item_action/chameleon/change/id
```

Comment:
drop the explicit type definition, byond will create it implicitly when you override the procs below